crates/nu-utils/standard_library/tests.nu

```diff
@@ -5,8 +5,8 @@ use std.nu *
 # `$in` must be a `record<file: string, module: string, name: string, pass: bool>`.
 #
 # the output would be like
-# - "<indentation> x <module>::<test>" all in red if failed
-# - "<indentation>   <module>::<test>" all in green if passed
+# - "<indentation> x <module>  <test>" all in red if failed
+# - "<indentation>   <module>  <test>" all in green if passed
```

Comment:
You used double spaces instead of one. If it is intentional, I have no problem with that.

Reply:
to be honest. it's not on purpose :eyes:

Reply:
fixed with the proper `<mod> <cmd>` notation in 3f3b9ca83ce5dfa863a1379abe2f7f6d841499c1 :yum:

Reply:
❤